IdentityCore/src/validation/MSIDB2CAuthority.m

```diff
@@ -165,4 +165,17 @@ + (NSURL *)normalizedAuthorityUrl:(NSURL *)url
     return [NSURL URLWithString:normalizedAuthorityUrl];
 }
 
+#pragma mark - NSCopying
+
+- (id)copyWithZone:(NSZone *)zone
+{
+    MSIDB2CAuthority *authority = [[self.class allocWithZone:zone] initWithURL:_url validateFormat:NO context:nil error:nil];
```

Comment:
Should we copy url that we pass?

Reply:
It seems like its related to above feedback which is addressed.

Reply:
nope, it should be  MSIDB2CAuthority *authority = [[self.class allocWithZone:zone] initWithURL:[_url copyWithZone:zone] validateFormat:NO context:nil error:nil];